modules/swagger-codegen/src/main/resources/LoopBackSwift/model.mustache

```diff
@@ -0,0 +1,46 @@
+{{#models}}{{#model}}import ObjectMapper
+{{#useRealm}}
+import Realm
+import RealmSwift
+{{/useRealm}}
+
+public class {{classname}} : {{#useRealm}}Object, {{/useRealm}}Mappable, CustomStringConvertible {
```

Comment:
Yes. It is possible to add comments using `{{! <comment> }}`. Which parts of a template do you suggest should be commented?